src/components/HeartbeatBar.vue

```diff
@@ -4,11 +4,17 @@
             <div
                 v-for="(beat, index) in shortBeatList"
                 :key="index"
-                class="beat"
-                :class="{ 'empty': (beat === 0), 'down': (beat.status === 0), 'pending': (beat.status === 2), 'maintenance': (beat.status === 3) }"
-                :style="beatStyle"
+                class="beat-hover-area"
+                :class="{ 'empty': (beat === 0) }"
```

Comment:
To prevent hovering when the beat is empty, I also added the `.empty` class to this.